Firestore/Example/Tests/GoogleTest/FSTGoogleTestTests.mm

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+#import <objc/runtime.h>
+
+#include "gtest/gtest.h"
+
+/**
+ * An XCTest test case that finds GoogleTest test cases, runs them, and generates results suitable
+ * for reporting failures back to Xcode. This allows tests written in C++ that don't rely on XCTest
+ * to coexist in this project.
+ *
+ * Each GoogleTest TestCase is mapped to a dynamically generated XCTestCase class. Each GoogleTest
+ * TEST() is mapped to a test method on that XCTestCase. An XCTestSuite is created for each
+ * TestCase and added to the suite returned by +defaultTestSuite.
+ */
+@interface FSTGoogleTestTests : XCTestCase
+@end
+
+namespace {
+
+/**
+ * Finds the TestInfo corresponding to the test method corresponding to the given test case class
+ * and selector. Fails the test and returns nullptr if the TestInfo cannot be found.
```

Comment:
"the TestInfo corresponding to the test method corresponding to the ..." is hard for me to make sense of.

Would it be sufficient to just say "Finds the TestInfo corresponding to the provided test case class and selector." ?

Reply:
This function has been removed, but I applied a similar fix to the next one.